include/grass/defs/raster.h

```diff
@@ -406,8 +406,8 @@ int Rast_maskfd(void);
     (*(const DCELL *)(dcellVal) != *(const DCELL *)(dcellVal))
 
 void Rast__set_null_value(void *, int, int, RASTER_MAP_TYPE);
-void Rast_set_null_value(void *, int, RASTER_MAP_TYPE);
-void Rast_set_c_null_value(CELL *, int);
+void Rast_set_null_value(void *, unsigned int, RASTER_MAP_TYPE);
+void Rast_set_c_null_value(CELL *, unsigned int);
```

Comment:
`unsigned int` is not large enough (below you use size_t), and since `Rast_set_null_value()` calls `Rast_set_[c|f|d]_null_value()`, the defs must all be the same.